Fix: Complete rewrite

isabelroses opened the pull request:
This is a complete YouTube rewrite for the themes and documentation. 

I am willing to help maintain it too if it is necessary.

Commit:
button changes

isabelroses committed:
Update YouTubeCatppuccinMocha.user.css

isabelroses committed:
Update YouTubeCatppuccinMacchiato.user.css

isabelroses committed:
Update YouTubeCatppuccinLatte.user.css

isabelroses committed:
Update YouTubeCatppuccinMacchiato.user.css

isabelroses committed:
Update YouTubeCatppuccinFrappe.user.css

isabelroses committed:
Update README.md

isabelroses committed:
Update README.md

Commit:
button text looks nicer

Commit:
search color + grammer

Commit:
fix theme name

isabelroses committed:
Update YouTubeCatppuccinFrappe.user.css

isabelroses committed:
Update YouTubeCatppuccinLatte.user.css

isabelroses committed:
Update YouTubeCatppuccinMacchiato.user.css

isabelroses committed:
vers

isabelroses committed:
vers

isabelroses committed:
vers

isabelroses committed:
docs and images

Commit:
removed images

Commit:
fixed buttons and toggles

Commit:
fixed overflow

nullishamy commented on src/YouTubeCatppuccinFrappe.user.css:
```diff
@@ -1,11 +1,11 @@
-@homepageURL https://github.com/catppuccin/YouTube
-@version 1.3.7
-@updateURL https://github.com/catppuccin/YouTube/raw/main/src/YouTubeCatppuccinFrappe.user.css
+@namespace github.com/isabelroses/YouTube
```
This should still link to ctp/youtube.

isabelroses replied:
I shall fix that right now

nullishamy commented on src/YouTubeCatppuccinFrappe.user.css:
```diff
@@ -4187,36 +3665,20 @@
     }
-
-    /*Fix like, dislike and share icons and text using wrong color*/
-    .yt-spec-button-shape-next__icon,
```
Why was this fix removed?

isabelroses replied:
this was a temporary fix and it now appears correctly.

nullishamy commented on src/YouTubeCatppuccinLatte.user.css:
```diff
@@ -1,11 +1,11 @@
-@homepageURL https://github.com/catppuccin/YouTube
-@version 1.3.7
-@updateURL https://github.com/catppuccin/YouTube/raw/main/src/YouTubeCatppuccinLatte.user.css
+@namespace github.com/isabelroses/YouTube
```
Same as above, should be ctp/youtube.

nullishamy commented on src/YouTubeCatppuccinMacchiato.user.css:
```diff
@@ -1,11 +1,11 @@
-@homepageURL https://github.com/catppuccin/YouTube
-@version 1.3.7
-@updateURL https://github.com/catppuccin/YouTube/raw/main/src/YouTubeCatppuccinMacchiato.user.css
+@namespace github.com/isabelroses/YouTube
```
Same as above, should be ctp/youtube.

nullishamy commented on src/YouTubeCatppuccinMocha.user.css:
```diff
@@ -1,11 +1,11 @@
-@homepageURL https://github.com/catppuccin/YouTube
-@version 1.3.7
-@updateURL https://github.com/catppuccin/YouTube/raw/main/src/YouTubeCatppuccinMocha.user.css
+@namespace github.com/isabelroses/YouTube
```
Same as above, should be ctp/youtube.

nullishamy commented on src/YouTubeCatppuccinLatte.user.css:
```diff
@@ -4187,36 +3662,20 @@
     }
-
-    /*Fix like, dislike and share icons and text using wrong color*/
-    .yt-spec-button-shape-next__icon,
```
Same as above, why was this fix removed?

isabelroses replied:
this was a temporary fix and it now appears correctly.
![image](https://user-images.githubusercontent.com/71222764/223218484-1524e36c-3eee-4987-8373-717e36cc0509.png)
example form mocha

nullishamy commented on src/YouTubeCatppuccinMacchiato.user.css:
```diff
@@ -4187,36 +3661,20 @@
     }
-
-    /*Fix like, dislike and share icons and text using wrong color*/
-    .yt-spec-button-shape-next__icon,
```
Same as above, why was this fix removed?

isabelroses replied:
this was a temporary fix and it now appears correctly.
![image](https://user-images.githubusercontent.com/71222764/223218484-1524e36c-3eee-4987-8373-717e36cc0509.png)
example form mocha

nullishamy commented on src/YouTubeCatppuccinMocha.user.css:
```diff
@@ -4187,36 +3661,20 @@
     }
-
-    /*Fix like, dislike and share icons and text using wrong color*/
-    .yt-spec-button-shape-next__icon,
```
Same as above, why was this fix removed?

isabelroses replied:
this was a temporary fix and it now appears correctly.
![image](https://user-images.githubusercontent.com/71222764/223218484-1524e36c-3eee-4987-8373-717e36cc0509.png)
example form mocha

Commit:
returned to ctp

isabelroses commented:
So I put all the ctp links back within the code. And for this section ```/*Fix like, dislike and share icons and text using wrong color*/
    .yt-spec-button-shape-next__icon, ```
I removed it as it was a temporary fix and will work in the newer version. The following example is from the mocha port showing it does work:
![image](https://user-images.githubusercontent.com/71222764/223218484-1524e36c-3eee-4987-8373-717e36cc0509.png)

Commit:
returned readme to ctp

Commit:
consitancy

Commit:
new youtube video links for description

elkrien commented:
> I am willing to help maintain it too if it is necessary.

Would be great because I have no time to be involved anymore. We will need to wait for organizations admin to add You as a maintainer (I have no rights to do it).
@nullishamy @nekowinston - maybe You can help?

nekowinston commented:
> > I am willing to help maintain it too if it is necessary.
> 
> Would be great because I have no time to be involved anymore. We will need to wait for organizations admin to add You as a maintainer (I have no rights to do it).

I've added @isabelroses as a maintainer. Thanks for letting us know @elkrien!

Commit:
reverting to older PR about comments

Commit:
hover buttons on player

Commit:
tooltip

Commit:
readablity of highlighted text

isabelroses commented:
PR's that this fixes:
[fix: style leaking to other sites](https://github.com/catppuccin/youtube/pull/30) - caused by brackets, now fixed
[fix: Notification icon text color](https://github.com/catppuccin/youtube/pull/28) - tests suggest this is fixed
[fix: Improve comment readability](https://github.com/catppuccin/youtube/pull/23) - This is an older PR that I reverted some of the code back to
[fix: subscribe button](https://github.com/catppuccin/youtube/pull/36) and [fix: subscribe button](https://github.com/catppuccin/youtube/pull/35) - both my own PRs however this was something completely rewritten in the change and significant testing into the matter with help from [lighttigerXIV](https://github.com/lighttigerXIV)
[chore: update buttons shape](https://github.com/catppuccin/youtube/pull/26) - another change for text color of subscribe and join button for readability fixed with this change again
[fix: some buttons and player tooltips](https://github.com/catppuccin/youtube/pull/22) - buttons were fixed during rewrite and I just committed the change for the tooltips.
[fix: metadata card](https://github.com/catppuccin/youtube/pull/3) - fixed in rewrite
[fix: selection readablity](https://github.com/catppuccin/youtube/pull/1) - newer iterations of the code override this, reverting back to this

Issues that remain:
[error pages](https://github.com/catppuccin/youtube/issues/11) - partially fixed

Commit:
prevent leak to VR

Commit:
concent EU laws causes changes which is diffrent per yt channel

Commit:
error pages

Commit:
flavor consistancy

Commit:
ctp consistancier

Commit:
dev branch

isabelroses committed:
fixed

isabelroses committed:
Merge pull request #1 from isabelroses/develop

dev branch

Commit:
editorconfig-checker

Commit:
Merge branch 'main' of https://github.com/isabelroses/youtube

Commit:
frappe fixes

Commit:
100% file consistancy

Commit:
formating undone beacuse it breaks it

nullishamy commented:
> PR's that this fixes:

You can write `Closes <pr>` to have GitHub automatically close the issues when this is merged btw.
